### DISCUSSION TITLE
Improve date format selection

### DISCUSSION THREAD

**Discussion by JoryHogeveen** [pods-framework/pods#6826] [Ideas]
https://wordpress.org/support/topic/a-suggestion-for-field-type/

> In the drop down list you wrote different formats of 08/07/2022
> I suggest if it is possible in your update replace “08” with a number bigger than “12”, to help user do not confuse.

To elaborate more.
We currently use the current date in this location. It might be more convenient in some cases to use a fixed date format?

**Comment by sc0ttkclark** [comment]
Love the idea of making this more clear and using a static date for those.

**Comment by JoryHogeveen** [comment]
@sc0ttkclark Any preference for the date we will use here?
Personally I like the current implementation more but I get that some people might be confused.

We could also start the list with the month name formats like "July 20th, 2022" so it's clear what date is being used.

And/or we could also visually add the format after the date.
Example:

- July 20th, 2022 (F j, Y)
- 20-07-2022 (d-m-Y)
- 07/20/2022 (m/d/Y)
- etc

**Comment by sc0ttkclark** [comment]
To make it totally easy for anyone and everyone, perhaps we should go with 12/31/2022 (always using current year)?

**Comment by sc0ttkclark** [comment]
I like your idea about adding the actual format too, that def should be done. With the format added, perhaps current date is fine too.
